docs/appendix/glossary/pin-number.md

```diff
@@ -2,7 +2,7 @@
 title: Pin Number
 id: pin-number
 full_link:
-short_description: A pin's number is the index of the pin on the board.
+short_description: A pin's number is the index of the pin on the board. Not the same as a pin's GPIO number.
```

Comment:
```suggestion
short_description: A pin number is the index of the pin on the board. Not the same as a pin's GPIO number.
```

Reply:
<img width="845" alt="Screenshot 2023-05-04 at 5 00 53 PM" src="https://user-images.githubusercontent.com/20797913/236328204-5a723f35-69e2-4fbc-997f-34463e7d3552.png">
omg-- I wasn't sure about what is better here and was going to comment asking but I decided to ask ChatGPT, and-- great explanation AI. Will make this change!